features/enlinkd/tests/src/test/java/org/opennms/netmgt/enlinkd/Nms4930EnIT.java

```diff
@@ -418,15 +419,15 @@ private void testNms4930Bft(final boolean disableBridgeVlanDiscovery) throws Exc
             assertNotNull(maclink.getBridgePort());
             assertNotNull(maclink.getNode());
             assertNotNull(maclink.getMacAddress());
-            System.err.println(maclink.toString());
+            System.err.println(maclink);
```

Comment:
why System.err?

Reply:
just because it writes red lines. During test check

Reply:
I used system.err everywhere in enlinkd tests.